contracts/src/v0.8/vrf/dev/TrustedBlockhashStore.sol

```diff
@@ -1,7 +1,7 @@
 // SPDX-License-Identifier: MIT
-pragma solidity 0.8.6;
+pragma solidity 0.8.19;
 
-import {ChainSpecificUtil} from "../../ChainSpecificUtil.sol";
+import {ChainSpecificUtil} from "./libraries/ChainSpecificUtil.sol";
```

Comment:
Same question as for comment [above](https://github.com/smartcontractkit/chainlink/pull/12479/files#r1530055990).